dao/src/main/scala/za/co/absa/enceladus/dao/EnceladusRestDAO.scala

```diff
@@ -35,7 +35,7 @@ import com.fasterxml.jackson.databind.SerializationFeature
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
 import com.fasterxml.jackson.module.scala.DefaultScalaModule
 import com.typesafe.config.ConfigFactory
-import sun.security.krb5.internal.ktab.KeyTab
+import sun.security.krb5.internal.ktab.KeyTab //scalastyle:ignore illegal.imports
```

Comment:
Why ignoring an illegal import?

Reply:
As it's the key part of the `KeyTab` support and imported with clear intent.
Having _warnings_ that have no consequence just adds noise. If the import is a worry, let's create a ticket for it, that might be eventually addressed.